ivy_tests/test_ivy/test_frontends/config/torch.py

```diff
@@ -75,3 +77,34 @@
     "complex128",
 ]
 invalid_complex_dtypes = []
+
+
+# Helpers for function testing
+
+
+Dtype = torch.dtype
+Device = torch.device
+
+
+def native_array(x):
+    return torch.tensor(x)
+
+
+def is_native_array(x):
+    return isinstance(x, (torch.Tensor, torch.nn.Parameter))
+
+
+def to_numpy(x):
+    return x.numpy()
+
+
+def as_native_dtype(dtype: str):
+    return torch.tensor([], dtype=dtype).dtype
```

Comment:
This fails.

Reply:
Well, is there a way we can retrieve a torch dtype from its string? I think we need to manually hardcode that